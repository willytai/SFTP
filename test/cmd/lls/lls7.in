lls -l lls/dirforcheck -h
