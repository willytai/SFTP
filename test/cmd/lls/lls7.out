sftp> lls -l lls/dirforcheck -h
-r--r--r--.  1 willytai staff  143B Oct 22 23:42 file2.txt
-r--r--r--.  1 willytai staff    7B Oct 22 23:42 file3.txt
-r--r--r--.  1 willytai staff    2B Oct 22 23:42 file1.txt
sftp> 
